xrayutilities/materials/cif.py

```diff
@@ -237,6 +261,9 @@ def SymStruct(self):
                         unique = False
                 if unique:
                     unique_pos.append(pos)
+            for o,r in zip(('+','-'),('p','m')):
+                if o in el:
+                    el = el.replace(o,r)
```

Comment:
I think this i in the meantime also implemented. generally the cif parser is now more powerful than it was a year ago

Reply:
Neat!